clients/drcachesim/tools/opcode_mix.h

```diff
@@ -107,6 +127,13 @@ class opcode_mix_t : public analysis_tool_t {
         uint category;
     };
 
+    class snapshot_t : public interval_state_snapshot_t {
+    public:
+        int64_t instr_count_;
```

Comment:
It would help to clarify that these counts are cumulative when generated, and converted to deltas when finalizing.

Reply:
Done.